test/unit/lib/plugins/aws/deploy-function.test.js

```diff
@@ -636,6 +791,46 @@ describe('test/unit/lib/plugins/aws/deployFunction.test.js', () => {
       },
       Layers: [layerArn, secondLayerArn],
     });
+
+    // Remove all arn layers
+    runServerlessConfig.options.function = 'noLayers';
+    await runServerless(runServerlessConfig);
+    expect(updateFunctionConfigurationStub).to.be.calledWithExactly({
+      FunctionName: 'noLayers',
+      Layers: [],
+    });
+
+    // Add layer and environment variables when console layer is present
+    runServerlessConfig.options.function = 'consoleAddLayerAndEnvironment';
+    await runServerless(runServerlessConfig);
+    expect(updateFunctionConfigurationStub).to.be.calledWithExactly({
+      FunctionName: 'consoleAddLayerAndEnvironment',
+      Environment: {
+        Variables: {
+          ANOTHERVAR: 'anothervalue',
+          VARIABLE: 'value',
+          VARIABLE2: 'value2',
+          AWS_LAMBDA_EXEC_WRAPPER: '/opt/lib/libthundra-wrapper.so',
+          SLS_ORG_ID: '123',
+        },
+      },
+      Layers: [layerArn, secondLayerArn, consoleLayerArn],
+    });
+
+    // Remove all arn layers and remove environment variable when console layer is present
+    runServerlessConfig.options.function = 'consoleRemoveLayerAndEnvironment';
+    await runServerless(runServerlessConfig);
```

Comment:
Each `runServerless` run ideally should configured as different desriptive `it` statements

It makes sense to join under one `it` if we're confirming on some processing flow where step B needs to be preceded by step A, but here I believe it's not the case.

Reply:
Okay so we basically want what we had before [I reverted this](https://github.com/serverless/serverless/pull/11849#discussion_r1146315205) just without the `describe` blocks?

Reply:
Actually sorry, maybe it is more like [the tests](https://github.com/serverless/serverless/pull/11849/commits/7b123e8c8c7be8e658f907786c947f5ca5fc39f3#diff-edf4f6791efd504c27e0dd20e05161d00333371eea770dfb12247e0170f3403b) I had set up originally, right?